Freshly generated generator failed npm test

If I generate a generator with the latest version of the generator it failes eslint
```
$ npm test

> generator-development@0.0.0 pretest /Users/jtomaw/Development/generator-development
> eslint .


/Users/jtomaw/Development/generator-development/__tests__/app.js
   8:19  error  Insert `⏎······`                                              prettier/prettier
   9:21  error  Replace `someAnswer:·true` with `·someAnswer:·true·`          prettier/prettier
  13:18  error  Replace `⏎······'dummyfile.txt'⏎····` with `'dummyfile.txt'`  prettier/prettier

/Users/jtomaw/Development/generator-development/generators/app/index.js
   9:14  error  Insert `⏎······`                  prettier/prettier
  10:1   error  Insert `··`                       prettier/prettier
  11:5   error  Replace `)` with `··)⏎····`       prettier/prettier
  13:22  error  Insert `⏎······`                  prettier/prettier
  14:1   error  Replace `······` with `········`  prettier/prettier
  15:1   error  Insert `··`                       prettier/prettier
  16:1   error  Insert `··`                       prettier/prettier
  17:7   error  Insert `··`                       prettier/prettier
  18:5   error  Replace `}` with `··}⏎····`       prettier/prettier

✖ 12 problems (12 errors, 0 warnings)
  12 errors, 0 warnings potentially fixable with the `--fix` option.

npm ERR! Test failed.  See above for more details.
```

--- a/subgenerator/templates/index.js
+++ b/subgenerator/templates/index.js
@@ -7,11 +7,7 @@ module.exports = class extends Generator {
   prompting() {
     // Have Yeoman greet the user.
     this.log(
-      yosay(
-        'Welcome to the <%- superb %> ' +
-          chalk.red('<%= generatorName %>') +
-          ' generator!'
-      )
+      yosay(`Welcome to the <%- superb %> ${chalk.red('<%= generatorName %>')} generator!`)
     );
 
     const prompts = [

